### ISSUE TITLE
Redirect Issue: forced ssl

### ISSUE THREAD

**Issue by GeneralLuzi** [jc21/nginx-proxy-manager#63]
Hi,

the following szenario I would like to achive:
All of the following should point to the same URL https://www.subdomain.domain.tld:
- https://www.subdomain.domain.tld
- http://www.subdomain.domain.tld
- https://subdomain.domain.tld
- http://subdomain.domain.tld

**1) Szenario Proxy Host:**
1a) Szenario: https://www.subdomain.domain.tld   --> create Reverse Proxy Host inkl. SSL
1b) Szenario: http://www.subdomain.domain.tld    --> without ssl --> Update Reverse Proxy from 1a) to Force SSL

**2) Szenario Redirect Host**
2a) Szenario: http://subdomain.domain.tld      -->create Redirect Host (point to URL from 1a)
2b) Szenario: https://subdomain.domain.tld    --> update Redirect Host from 2a) to SSL + Force SSL

**Testing:**
1a) works as specified
1b) works as specified
2a) does not work: url is redirected to https://http//subdomain.domain.tld
2b) does not work - like 2a) 

If if I try to disable "Force SSL" within the Redirect Host but it does not save this new setting. (the same as specifed for http/2 in #61 --> perhaps there should be a new issue opened?!)

I deleted the Redirect Host from 2) and created the a new one with SSL settings, but without "Forced SSL".  Same picture. 

Can some verify my observation. Or do I missunderstand something how to use this manager?

Thanks a lot!

**Comment by GeneralLuzi** [comment]
update:
the relevant conf within redirection_host folder has the following specified:
return 301 
- $scheme://https://www.sudomain.domain.tld$request_uri;

This migt cause this issue!

**Comment by jc21** [comment]
Regarding the location `https://http//subdomain.domain.tld`, I just tested myself and couldn't replicate this outcome.

TL;DR for below: please try again with the latest docker image.


To simplify the setup required, you just want to
1. create a Redirection Host to from source `subdomain.domain.tld` to destination `www.subdomain.domain.tld`
2. create a Proxy Host from `www.subdomain.domain.tld` to upstream server.

When it comes to redirecting and SSL, when the destination http server supports both HTTP and HTTPS connections (as a Proxy Host does), your Redirection Host doesn't need to Force SSL because you'll want to leave that determination up to the destination. However if the destination server only supported HTTPS then you'd want to force SSL before it even got to the destination.

A redirection host will forward to the destination on the same protocol scheme requested, and if force SSL is on then it will redirect to that first.

Btw, when I say that the server supports HTTP and HTTPS I mean from an open port point of view, not from a Proxy Host configured to forward both point of view.

**Comment by mackcoding** [comment]
I am having this issue as well. I want to redirect to an https but I get https://http//url. The URL is an IP address that requires https.

**Comment by jc21** [comment]
I'm going to need to see the configuration for these hosts.

1. Screenshot the edit proxy host dialog, with the domain name removed for privacy
2. Find the nginx configuration for that host in your `data/nginx/proxy_host` folder and attach that here, changing the domain name for privacy

**Comment by toupou** [comment]
I still have the same issue. forwarding to another domain adds https//www. 

happy to provide any config files if needed

**Comment by rockpham** [comment]
> I am having this issue as well. I want to redirect to an https but I get https://http//url. The URL is an IP address that requires https.

I am also having this issue with redirection hosts. Attached are some screenshots, I hope that it's useful.
Please kindly help!

![CleanShot 2023-0105 at 19 55 42@2x](https://user-images.githubusercontent.com/6811407/210786211-f003c536-a57a-48d2-8f25-1efe28b20048.png)
![CleanShot 2023-0105 at 19 55 48@2x](https://user-images.githubusercontent.com/6811407/210786236-86629714-c5af-4685-994e-79d6f18156c9.png)
![CleanShot 2023-0105 at 19 57 01@2x](https://user-images.githubusercontent.com/6811407/210786246-42b4980e-a3f8-437c-8fea-75d7068b1a0f.png)

**Comment by Cruv** [comment]
I know this is a zombie issue but I am also running into this with the same exact result as the poster above me.

**Comment by rockpham** [comment]
I got the redirection host working by using 301, instead of 308

<img width="491" alt="ảnh" src="https://user-images.githubusercontent.com/6811407/220574755-982fa1fc-fc00-4114-a18b-fa992d28450e.png">

**Comment by DerPW** [comment]
This issue is still happening. (v2.9.21)

**Comment by Barina** [comment]
Issue still persist in v2.11.3 using 308 still getting double https:// on a redirection host.

**Comment by BaconFist** [comment]
Issue persists in 2.12.1

**Comment by Chaxiraxi** [comment]
As discussed in [this issue](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3950), I think this is not possible with Nginx Proxy Manager. See [this reddit post](https://www.reddit.com/r/nginxproxymanager/comments/pnec6x/comment/henl8vx/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
